_pages/homepage.md

```diff
@@ -53,93 +53,144 @@ flow:
-            bare-metal configuration.
+            Bare-metal development is supported by exit from the firmware stack at two points:
+            * Post SCP execution: System level IP is initialised, development is possible from Application processor reset. Supports true bare-metal scenarios.
+            * Post TF-A execution: Lead Application processor is initialised and runtime services are available. Supports ports of new RTOS environments and more complex bare-metal workloads
 
-            ### AArch64 Linux CHERI LLVM/Clang toolchain
+            Please refer to `standalone-baremetal-readme.rst` in the Documentation repository.
 
-            This toolchain is an experimental AArch64 hosted variant, primarily intended to be used together
-            with Arm's [Morello Instruction Emulator](https://developer.arm.com/architectures/cpu-architecture/a-profile/morello/development-tools#instruction-emulator). It includes a C/C++ compiler (clang), linker (lld), various
-            utilities and run-time libraries, but does not include a C library. See this [knowledge base article](https://developer.arm.com/documentation/ka002048/1-0/?lang=en) for
-            more details.
-      - format: text
-        style: bg-white p-4 my-3
-        text_content:
-          text: |-
-            ## Firmware
+            [Arm Development Studio Morello Edition](https://developer.arm.com/architectures/cpu-architecture/a-profile/morello/development-tools#arm-development-studio) provides a development environment for bare-metal configurations and includes a "hello world" example.
 
-            The Development Platform is supported by Morello aware ports of standard open source firmware
-            components: [SCP firmware](https://github.com/ARM-software/SCP-firmware), [Trusted Firmware TF-A](https://www.trustedfirmware.org/), [UEFI EDK2](https://github.com/tianocore/edk2)
-      - format: text
-        style: bg-white p-4 my-3
-        text_content:
-          text: |-
-            ## Morello Linux Kernel
+            ## Android Enablement
 
-            Modified ACK (Android common Kernel).
-      - format: text
-        style: bg-white p-4 my-3
-        text_content:
-          text: |-
-            ## Nano Android
+            An evolving Android environment has been available for Morello since the first release in October 2020. 
 
-            An integrated Android stack supporting a reduced nano profile is hosted in the Morello code repositories.
+            This includes a minimal (nano) headless system Android (64bit) profile suitable for use with the FVP. Full Android boot will be supported on the Morello hardware platform
+
+            Support for Pure capability (Pure-cap) applications (along with several example ports) is provided by a Morello ACK (Android Common Kernel) and Bionic library variants built using the CHERI LLVM/Clang toolchain.
```

Comment:
The abbreviated form is purecap not pure-cap. Pure should not be capitalised.

Reply:
To explain the odd approach this is Kyle's PR but my copy - so I'll work though & confirm each comment.

Ack. We'll align with purecap throughout copy. Capitalising only if it starts the sentence.